ignite/pkg/chaincmd/chaincmd.go

```diff
@@ -486,6 +501,11 @@ func (c ChainCmd) BankSendCommand(fromAddress, toAddress, amount string) step.Op
 		optionYes,
 	)
 
+	// Apply the options provided by the user
+	for _, applyOption := range options {
+		command = applyOption(command)
+	}
```

Comment:
Nit, normally we use `apply` when using this pattern:

```suggestion
	for _, apply := range options {
		command = apply(command)
	}
```

There are also cases where just `o` is used. I would recommend to settle on one case, in which case I would be on favor of `apply`.

Reply:
Take a look to the entire package. It uses `applyOption`. I just decided to stick with the pattern around.

Reply:
Sure I can homogenize everything in this PR.

Reply:
Done. Please check.